packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/legacy/constants.js

```diff
@@ -0,0 +1,5 @@
+export const DATE = '20/03/20';
```

Comment:
Shouldn't it be dynamic ? or returned by some API ?

Reply:
This is a hard-coded date as per functional spec. We have to tell customers with the legacy backup that there are new backup offers available form 20/03/20 and they must switch to one of these new backup offers. This date does not come from API.